presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1094,7 +1094,8 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
                 views.put(schemaTableName, new ConnectorViewDefinition(
                         schemaTableName,
                         Optional.ofNullable(table.get().getOwner()),
-                        decodeViewData(table.get().getViewOriginalText().get())));
+                        decodeViewData(table.get().getViewOriginalText().get()),
```

Comment:
Why is this needed here?

Reply:
This is the original behavior. When creating a view in HiveMetadata, the view data is encoded `setViewOriginalText(Optional.of(encodeViewData(viewData)))` ; so that here it needs to be decoded.

Reply:
Oops I miss matched the parenthesis and understood it wrong...